pint_server/app.py

```diff
@@ -494,15 +500,15 @@ def query_deletedby_images_in_provider_region(
                 PROVIDER_IMAGES_MODEL_MAP[provider].region == region,
                 PROVIDER_IMAGES_MODEL_MAP[provider].state == ImageState.deprecated,
                 PROVIDER_IMAGES_MODEL_MAP[provider].deprecatedon < deprecatedby,
-            )
+            ).order_by(desc(PROVIDER_IMAGES_MODEL_MAP[provider].publishedon)).all()
```

Comment:
Since this is querying deleted images, should we sort by `deletedon` in ascending order?

Reply:
if we're trimming by the publishedon date, we should sort by the publishedon date

Reply:
We are not trimming this one, yet.